stubs/pyjks/jks/jks.pyi

```diff
@@ -0,0 +1,107 @@
+from _typeshed import Incomplete
+from collections.abc import Iterable
+from typing import Any, NoReturn
+from typing_extensions import Final, Literal, Self, TypeAlias
+
+from .util import AbstractKeystore, AbstractKeystoreEntry
+
+__version_info__: tuple[int, int, int] | tuple[int, int, int, str]
+__version__: str
```

Comment:
Would make sense to add `Final[]` to these too, right? And for the `Literal[]` contstants?

Reply:
> Would make sense to add `Final[]` to these too, right?

Sure

> And for the Literal[] contstants?

I have a preference for `FOO: Final = 42` over `Foo: Final[Literal[42]]` (we allow both)

Reply:
Ah cool, changed

Reply:
FYI `FOO: Final = 42` isn't supported by the PyCharm type system. But to be fair, that's the least of its problems 😆

![image](https://github.com/python/typeshed/assets/137616/35fc23ba-2cb3-4cf6-a63b-227d7cbf14d3)

Reply:
uhhh yeah don't use PyCharm's type checker :)